api/Document.json

```diff
@@ -197,10 +197,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/alinkColor",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "64",
+              "notes": "Moved from <code>HTMLDocument</code>."
```

Comment:
What do these `HTMLDocument` notes mean exactly? After taking a closer look, I got the impression that `HTMLDocument` and `Document` are two names for the same thing, but on Chrome before version 64, these features were only available as part of `HTMLDocument` and not `Document`? Do I have the story right? If it's right, I'd appreciate an expansion of the note (e.g., "Before version 64, this was only available via the HTMLDocument API" or something like that, depending on whether I understood correctly)